src/OmniSharp/app.config

```diff
@@ -3,4 +3,20 @@
   <startup>
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6" />
   </startup>
+  <runtime>
```

Comment:
This will need to be updated everytime we update Roslyn to a new version (or, at least, everytime the assembly version changes). Correct?